### ISSUE TITLE
How to set up unread notifications 

### ISSUE THREAD

**Issue by musicfunfan** [dunst-project/dunst#1125]
Hello this is not a bug is a question i have. 
I want to display in my bar (Polybar) the number of new notifications that i do not have react with them. 
I fount the command  ``dunstctl count`` But this do not have category i want.... 
My question is... Can i have functionality with dunst ? 

I just need to know the number of the new notifications i have....

**Comment by fwsmit** [comment]
This functionality is not implemented in dunst.

**Comment by fwsmit** [comment]
Yep, if you want to have a proper count of unread notifications (and a way to bring them back), you'll have to implement that :)

**Comment by musicfunfan** [comment]
Copy thenx for the info... Can i mark that post something like feature request or something like that... ? Or there is no way this will be added ?

**Comment by fwsmit** [comment]
Done. You can edit the title to make it more of a request

**Comment by musicfunfan** [comment]
Ok thenx, I do not know much of c... If this was written on Python or Java i would try to add it my self... I will check the code anyway to see if i can make something happened. But i do not thing i know c that good... sadly...

**Comment by bynect** [comment]
If you want to get the number of waiting notification there is `dunstctl count waiting`.

Or do you want to get the number of notifications that you didn't interact with?

**Comment by musicfunfan** [comment]
The latter one i want to get number of notification that i did not open yet.... Like "unread notification". So if a notification come when i am not on the machine. I can see that for example i have 3 new notifications.

**Comment by bynect** [comment]
@fwsmit do we have a way to check if the user was idle when the notification timed out?

**Comment by musicfunfan** [comment]
All i want is to have a counter of notification on my bar. That i will clear when i am done reading the notifications. The idle part sound nice as feature but some times i have pause or mute notifications. When i am on a call and sharing screen for example so... I would not be idle.

**Comment by bynect** [comment]
> All i want is to have a counter of notification on my bar. That i will clear when i am done reading the notifications. The idle part sound nice as feature but some times i have pause or mute notifications. When i am on a call and sharing screen for example so... I would not be idle. 

If you want to only see the notification that are not yet shown you can use dunstctl count waiting

This will list also the paused notification and things in line

**Comment by fedang** [comment]
dunstctl count waiting does indeed return the number of undisplayed notifications:

```
for i in {0..100} ; do dunstify $i; done

dunstctl count waiting
92
```

assuming 8 is the limit at a time
